.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
Oh sure, what reads this? Eclipse et al? that's fine, those settings look standard and like what the project uses.

Reply:
eclipse needs a plugin, but some others support it by default: http://editorconfig.org/